.github/workflows/test.yml

```diff
@@ -52,6 +52,19 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y libpcap-dev
+        sudo apt-get upgrade -y
+    - name: Install Cassandra CPP driver
+      if: ${{ matrix.runner == 'ubuntu-18.04' }}
+      run: |
+        echo CPP version number
+        cc --version
+        cat /etc/lsb-release
+        wget https://downloads.datastax.com/cpp-driver//ubuntu/18.04/cassandra/v2.16.0/cassandra-cpp-driver_2.16.0-1_amd64.deb &
+        wget https://downloads.datastax.com/cpp-driver/ubuntu/18.04/cassandra/v2.16.0/cassandra-cpp-driver-dev_2.16.0-1_amd64.deb &
+        wget https://downloads.datastax.com/cpp-driver/ubuntu/18.04/dependencies/libuv/v1.35.0/libuv1_1.35.0-1_amd64.deb &
+        wget https://downloads.datastax.com/cpp-driver/ubuntu/18.04/dependencies/libuv/v1.35.0/libuv1-dev_1.35.0-1_amd64.deb &
+        wait
+        sudo apt -y install ./cassandra-cpp-driver_2.16.0-1_amd64.deb ./cassandra-cpp-driver-dev_2.16.0-1_amd64.deb ./libuv1_1.35.0-1_amd64.deb ./libuv1-dev_1.35.0-1_amd64.deb
```

Comment:
Can we add to the Contributing docs that you need to install `cassandra-cpp-driver`?

Reply:
And to `install_ubuntu_packages.sh` when #368 gets merged